File: simple_merge.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> simple_merge(vector<int> &a, vector<int> &b)
{
    int total, j, k;

    total = a.size() + b.size();
    j = 0;
    k = 0;

    vector<int> c(total);

    a.push_back(INT_MAX);
    b.push_back(INT_MAX);

    for (int i = 0; i < total; i++)
    {
        c[i] = (a[j] < b[k]) ? a[j++] : b[k++];
    }

    a.erase(a.end() - 1);
    b.erase(b.end() - 1);

    return c;
}

int main()
{
    int n1, n2, x;
    vector<int> a, b;

    scanf("%d%d", &n1, &n2);

    for (int i = 0; i < n1; ++i)
    {
        scanf("%d", &x);
        a.push_back(x);
    }

    for (int i = 0; i < n2; ++i)
    {
        scanf("%d", &x);
        b.push_back(x);
    }

    vector<int> result;

    result = simple_merge(a, b);

    for (int i = 0; i < result.size(); ++i)
    {
        printf("%d\n", result[i]);
    }

    return 0;
}

File: simple_merge.py
def simpleMerge(a, b):
    total = len(a) + len(b)
    j, k = 0, 0

    c = list()

    for i in range(total):
        if (k == len(b) or (j < len(a) and a[j] < b[k])):
            c.append(a[j])
            j += 1
        else:
            c.append(b[k])
            k += 1

    return(c)


aSize, bSize = map(int, input().split()) # Apenas para acordar com o simple_merge.in

a = list(map(int, input().split())) # Lê uma lista em uma linha
b = list(map(int, input().split())) # Lê uma lista em uma linha

result = simpleMerge(a, b)

for i in result:
    print(i)
